src/test/java/com/java_academy/AppTest.java

```diff
@@ -0,0 +1,7 @@
+package com.java_academy;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest{
```

Comment:
Use auto formatting for whole code - one time you have opening brace in the same line and next time you have it in the line below